src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -13,9 +13,10 @@ public class CliSyntax {
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_ID = new Prefix("i/");
     public static final Prefix PREFIX_HANDLE = new Prefix("h/");
-    public static final Prefix PREFIX_STUDENT_INFO = new Prefix("s/");
+    public static final Prefix PREFIX_STUDENT_INFO = new Prefix("m/");
```

Comment:
Should you reuse PREFIX_MODULE_CODE instead since both PREFIX_STUDENT_INFO and PREFIX_MODULE_CODE both refer to the same thing (module code)?

Reply:
Thanks updated!